src/components/Post.js

```diff
@@ -63,7 +77,7 @@ const PostContainer = styled.div`
       font-size: 19px;
       font-family: "Lato", sans-serif;
     }
-    > p {
+    > p > span {
       color: #b7b7b7;
       font-size: 17px;
       font-family: "Lato", sans-serif;
```

Comment:
You can remove this line, because this font-family was already declared on reset.

Reply:
Oh sorry, I merged it before seeing this comment. I will fix it in the next pull request